ajax/libs/inheritance.js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "inheritance.js",
+  "npmName": "inheritance-js",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "*.js"
+      ]
+    }
+  ],
+  "filename": "inheritance.min.js",
+  "version": "0.1.2",
+  "description": "A simple and small implementation of object inheritance using pure JavaScript.",
+  "homepage": "https://github.com/bsara/inheritance.js#readme",
+  "keywords": [
+    "class",
+    "classes",
+    "inherit",
+    "inheritjs",
+    "inherit-js",
+    "inherit.js",
+    "inheritance",
+    "inheritancejs",
+    "inheritance-js",
+    "inheritance.js",
+    "object",
+    "objects",
+    "prototype"
+  ],
+  "maintainers": [

```

Comment:
@bsara We don't need `"maintainers"` field but we can use `author` instead of it.
```json
  "author": {
    "name": "Brandon Sara (bsara)",
    "url": "http://bsara.github.io/"
  }
```